client/post-editor/editor-title/index.jsx

```diff
@@ -101,19 +104,36 @@ export default React.createClass( {
 				}
 				<TrackInputChanges onNewValue={ this.recordChangeStats }>
 					<TextareaAutosize
-						{ ...omit( this.props, Object.keys( this.constructor.propTypes ) ) }
+						{ ...ownProps }
 						className="editor-title__input"
-						placeholder={ this.translate( 'Title' ) }
+						placeholder={ translate( 'Title' ) }
 						onChange={ this.onChange }
 						onInput={ this.resizeAfterNewlineInput }
 						onBlur={ this.onBlur }
 						autoFocus={ isNew && ! isMobile() }
 						value={ post ? post.title : '' }
-						aria-label={ this.translate( 'Edit title' ) }
+						aria-label={ translate( 'Edit title' ) }
 						ref="titleInput"
 						rows="1" />
 				</TrackInputChanges>
 			</div>
 		);
 	}
-} );
+}
+
+export default connect(
+	( state, ownProps ) => {
+		const site = getSelectedSite( state );
+		const editedPostId = getEditorPostId( state );
+		const post = getEditedPost( state, site.ID, editedPostId );
+		const isNew = isEditorNewPost( state );
+
+		return {
+			isNew,
+			post,
+			site,
+			ownProps
```

Comment:
Yes, you are right! my answer was not so clear. Let me explain more. the `PostEditor` component renders the `EditorTitle` Component like this :  `<EditorTitle tabIndex="1" />` and the `EditorTitle` Component renders a textarea close to this : `<Textarea {...props />`.

The issue here, is to find the best way to compute the props that needs to be provided to the `Textarea`. In the example here, these props are `{ tabIndex: '1' }`. We need to find the best way to find these exact props without passing any extra prop provided by the HoC components used on the `EditorTitle`.

The options are : 
 1- `const props = omit( this.props, Object.keys( this.constructor.propTypes ) )` and make sure we define all the propTypes provided by HoC like localize even If we don't use them like the `moment` prop. Also If the localize HoC get updated and provides an extra prop, the `EditorTitle` component will have to be updated to "omit" the new prop.

 2- using `ownProps` since we know these are the only props provided to the `EditorTitle` container and not computed by any HoC.

 2- avoid using `{ ...props }` and use only something like `tabIndex={this.props.tabIndex}`. This approach is fine but we change the behavior of the `EditorTitle` component since we can not provide extra props (for example style={someStyle} or whatever) to the textarea

I hope this is clearer.